odoo/models.py

```diff
@@ -908,9 +908,9 @@ def splittor(rs):
 
                             if name == 'id':
                                 xml_ids = [xid for _, xid in value.__ensure_xml_id()]
-                                current[index] = ','.join(xml_ids) or False
+                                current[index] = ','.join(xml_ids)
                             else:
-                                current[index] = field.convert_to_export(value, record) or False
+                                current[index] = field.convert_to_export(value, record)
```

Comment:
Not sure about that one, this is for the subfields (e.g. `foo/.id`, `foo/name`, ...)

Reply:
But then this test fails for m2m https://github.com/odoo/odoo/blob/b8f02dd14d51e3b01eb85a784c9de5aa89cc4677/odoo/addons/test_impex/tests/test_export.py#L595-L598

Should I revert this test ?

Reply:
I just tested with the `res.partner` model and 'Tags/Tag Name'. and it causes the same issue. So we should probably export those as an empty string also. What's your opinion ?